pkg/execute/executor.go

```diff
@@ -144,10 +142,27 @@ func runKubectlCommand(args []string, clusterName string, isAuthChannel bool) st
 		if arg == AbbrWatchFlag.String() || strings.HasPrefix(arg, WatchFlag.String()) {
 			continue
 		}
+		// Check --cluster-name flag
 		if strings.HasPrefix(arg, ClusterFlag.String()) {
+			// Check if flag value is in current or next argument
 			if arg == ClusterFlag.String() {
+				if index == len(args)-1 {
+					return ""
+				}
+				arg = args[index+1]
```

Comment:
Can we avoid reassigning arg? The above `for loop` already takes care of it. Also rename `checkFlag` to something more meaningful like `isClusterFlagPresent`? `continue` here and process the value of `--cluster-name` in the next iteration.

Reply:
Made required changes and commited.